CONTRIBUTING.md

```diff
@@ -0,0 +1,604 @@
+
+![image](readme-images/xcode-tests.png?raw=true)
+
+
+## Debugging
+
+### Running Cocoapods Locally
+
+Framework development is quite different from typical app development when it comes to debugging and being able to test your code. First you will want to create a new app that uses the pods you changed. For instance, if you changed something in the `AWSS3TransferUtility` to test to see if it works, you will need to create a new app and run `pod init` and edit the `Podfile` to use the local version of the SDK you have. This is done by editing the `Podfile` of the new application to take a path to the pod instead of a version number like below:
+
+```ruby
+
+target 'MySampleApp' do
+  
+  use_frameworks!
+  # with :path I can direct my project to use a local path for the pod
+  pod 'AWSCore', :path => '~/Projects/aws-sdk-ios'      # 
+  pod 'AWSS3TransferUtility', :path => '~/Projects/aws-sdk-ios'           
+  
+end
+```
+Then you want to run `pod update` at the root of your sample app to make sure it is using the local version of the pods. When you open the workspace, under the Pods project, then you'll see a folder called Development Pods. This where our pods under local development were installed and if you want to change code here to debug, it will change in the other repo, just make sure you clean and re-build before testing your code again.
+
+## Pull Requests
+
+### Pull Request Checklist
+- Testing
+    - Unit test added or updated
+    - Integration test added or updated
+- Changelog
+    - Update Changelog with your updates in accordance with our pattern under the new version. Example below:
+
+    ```markdown
+    ## 2.xx.x
+
+    ### Bug Fixes
+
+    - **Breaking Change** List out any breaking changes here
+
+    - **Amazon S3 or name of service you changed**
+      - for example, if you made a change to S3, add the heading like above and list the bug fix here with the issue number.
+
+    ### Misc. Updates
+
+    -  Misc updates go here. Usually model changes
+    ```
+
+### Step 1: Open Issue
+
+If there isn't one already, open an issue describing what you intend to contribute. It's useful to communicate in advance, because sometimes, someone is already working in this space, so maybe it's worth collaborating with them instead of duplicating the efforts.
+
+### Step 2: Design (optional)
+
+In some cases, it is useful to seek for feedback on the design of your planned implementation. This is useful when you plan a big change or feature, or you want advice on what would be the best path forward.
+
+The GitHub issue is sufficient for such discussions, and can be sufficient to get clarity on what you plan to do. Make sure you tag any members of the AWS SDK team so we can help guide you: @kneekey23, @lawmicha, @wooj2, @palpatim, @royjit, @drochetti, @phani-srikar.
```

Comment:
common alias vs individual members: do we want to encourage tagging a common alias vs individual members? else we may have to keep updating the list of members as and when needed?

Reply:
problem here is that we dont have a common alias that is just iOS. it would tag the entire amplify org which covers js and android team :) not sure what else to do here although it's a valid point?

Reply:
https://github.com/orgs/aws-amplify/new-team !

Reply:
fixed